mite_http/__init__.py

```diff
@@ -4,7 +4,7 @@
 from contextlib import asynccontextmanager
 from functools import wraps
 
-from acurl import EventLoop
+import acurl_ng
```

Comment:
Shouldn't import this at the toplevel, if you are importing it conditionally later in the file.

Reply:
Missed that, thanks 👍